dotcom-rendering/src/lib/useIsSignedIn.ts

```diff
@@ -0,0 +1,35 @@
+import { getCookie } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+
+async function doOktaMethodForSignIn() {
+	const identityModule = await import('./identity');
+	return identityModule.isSignedInWithOkta();
```

Comment:
Given we don't use anything else from the identity module, I wonder if we could:

```suggestion
	return (await import('./identity')).isSignedInWithOkta();
```